channels/chan_echolink.c

```diff
@@ -20,6 +20,9 @@
  * --------
  * 02/5/23 - Danny Lloyd, KB4MDD <kb4mdd@arrl.net>
  * corrected memory leaks
+ * --------
+ * 12/7/24 - Caleb, K4PHP <caleb.k4php@gmail.com>
```

Comment:
Generally speaking these types of comments are heavily discouraged nowadays. Authorship is shown in the commit history

Reply:
I followed your header format, fair assumption IMO; resolved